src/mpyl/cli/__init__.py

```diff
@@ -1,23 +1,13 @@
 """Command Line Interface parsing for MPyL"""
 
-import asyncio
-import importlib
 import logging
 import os
 from dataclasses import dataclass
-from importlib.metadata import version as version_meta
-from pathlib import Path
 from typing import Optional
 
-import click
-import requests
```

Comment:
Not sure if you already checked, but just in case: can this be removed from `Pipfile` ?